vignettes/daily-series.Rmd

```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup}
-library(trendecon)
-```
-
-
-## Intro
-
-Google search results are available on a daily, weekly or monthly frequency. As with individual samples, querying Google at a different frequency will lead to substantially different results.
-
-Our goal is to produce long daily time series, ideally from 2006. However, Google does not provide daily or weekly data for such a long time period. We circumvent the problem by applying a moving window of daily and weekly queries over the whole time period.
-
-This vingette describes the production processes of trendecon. The production function start with `proc_` and usually operate on the file system. That means that calling this functions will write to your disk.
-
+Once the combination of useful keywords is selected we might want to use them as economic indicators in production and display them on [trendecon.org](https://www.trendecon.org/). This vignette describes the steps for adding new indicators to the production processes of "Trendecon".
 
-## Directory structure
+## Directory structure ##
 
-The `proc_` functions write to the working directory. If you are unsure about your current working directory, use `getwd()`.
+The names of production functions within the `trendecon` package start with `proc_`.
+They usually operate on the file system which means that calling these functions will write files to disk.
+The `proc_` functions will save files within two folders: `data` and `raw`, which are located under the working directory (`getwd()`).
+`raw` folder holds all the data downloaded from Google, plus some transformations, such as seasonally adjusted or combined series.
```

Comment:
```suggestion
`raw` folder holds all the data downloaded from Google that are automatically transformed by the `trendecon`package. The transformations include seasonal adjustments or combining multiple keywords related to the indicator.
```

Reply:
I see! So there are the raw time series, and on top some with transformations if called. My bad. I think the sentence you had written is good.